xbmc/Application.cpp

```diff
@@ -805,11 +802,11 @@ bool CApplication::Initialize()
     CServiceBroker::RegisterTextureCache(std::make_shared<CTextureCache>());
 
     std::string defaultSkin = std::static_pointer_cast<const CSettingString>(setting)->GetDefault();
-    if (!LoadSkin(settings->GetString(CSettings::SETTING_LOOKANDFEEL_SKIN)))
+    if (!LoadSkin(settings->GetString(CSettings::SETTING_LOOKANDFEEL_SKIN),this,this))
```

Comment:
Shouldn't this (and line 809) be `CApplicationSkinHandling::LoadSkin` just like the `CApplicationSkinHandling::ReloadSkin` counterpart?

Reply:
In ReloadSkin I needed a wrapper in CApplication to pass down the callback pointers. I qualify the call there to clarify that I call a method from the parent class. As there is no such ambiguity to clarify here, I saw no reason to do so.

Reply:
i can add the using statement if you prefer? or do you want me to qualify all parent method calls?

Reply:
No preference or technical reason to request that to be honest. its only a bit more clear to me LoadSkin is not defined in Capp itself but on the other class. Easier to detect in the future when improving and moving stuff but no big deal